### DISCUSSION TITLE
Polysleek and LLC 3.5.1

### DISCUSSION THREAD

**Discussion by root9191** [JulianPrieber/littlelink-custom#361] [Questions]
Hi, so in the newest Version of LLC my Name gets fade out and fade in again. In the older Version the Name changed the Color.
Can you check this?

**Comment by JulianPrieber** [comment]
Sure, I'll look into it

**Reply by root9191** [reply]
Is this in Version 3.5.2 fixed?

**Reply by JulianPrieber** [reply]
Haven't found what's causing it yet

**Reply by JulianPrieber** [reply]
Fixed on the latest beta

**Reply by JulianPrieber** [reply]
If there aren't any other bugs, I'll issue a patch in the next couple of days

**Reply by root9191** [reply]
i haven't found any bugs for now 😅
